src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -93,6 +96,9 @@
 -lohPinningInterval/-lohpi: g_lohPinningInterval
 meaning every Nth LOH object survived will be pinned. 
 
+-pohPinningInterval/-pohpi:
+meaning every Nth POH object survived will be pinned.
```

Comment:
now I'm puzzled to see the pohpi arg since everything on POH is pinned... @VSadov?

Reply:
It is allowed to pin POH objects, so `-pohpi` this _can_ be implemented.

I do not think there will be a lot of scenarios using this though.  I can imagine there could be scenarios where we would want to compare scenarios where pinning still happens, but target objects are POH-allocated.

In short - it was easier to match LOH here, than argue that we might not need this. I do not think this will be very popular knob.